cypress/support/fragments/data_import/job_profiles/newJobProfile.js

```diff
@@ -140,6 +140,33 @@ export default {
     cy.expect(Pane('New job profile').find(Accordion('Overview')).find(HTML(including(secondActionProfileName))).exists());
   },
 
+  linkMatchAndThreeActionProfiles(matchProfileName, firstActionProfileName, secondActionProfileName, thirdActionProfileName, forMatchesOrder = 0) {
+    // link match profile to job profile
+    cy.get('[id="type-selector-dropdown-linker-root"]').click();
+    cy.do(matchButton.click());
+    ModalSelectActionProfile.searchActionProfileByName(matchProfileName, 'match');
+    ModalSelectActionProfile.selectActionProfile(matchProfileName, 'match');
+    cy.expect(Pane('New job profile').find(Accordion('Overview')).find(HTML(including(matchProfileName))).exists());
+    // link first action profile to match profile
+    cy.get('[id*="type-selector-dropdown-ROOT"]').eq(forMatchesOrder).click();
+    cy.do(actionsButton.click());
+    ModalSelectActionProfile.searchActionProfileByName(firstActionProfileName);
+    ModalSelectActionProfile.selectActionProfile(firstActionProfileName);
+    cy.expect(Pane('New job profile').find(Accordion('Overview')).find(HTML(including(firstActionProfileName))).exists());
+    // link second action profile to match profile
+    cy.get('[id*="type-selector-dropdown-ROOT"]').eq(forMatchesOrder).click();
```

Comment:
remoge selector to const

Reply:
no, only '[id*="type-selector-dropdown-ROOT"]' i mean

Reply:
I've created TODO for this changes